source/fundamentals/builders/aggregates.txt

```diff
@@ -537,3 +537,21 @@ The following example creates a pipeline stage that executes two parallel aggreg
    :end-before: // end facet
    :language: java
    :dedent:
+
+SetWindowFields
+---------------
+
+Use the ``setWindowFields()`` method to create a :manual:`$setWindowFields </reference/operator/aggregation/setWindowFields/>`
+pipeline stage that allows using window operators to perform operations
+on a specified span of documents in a collection.
```

Comment:
Do we want to also add a link to the `WindowedComputations` class, something like:

> The driver includes the [`WindowedComputations`](https://mongodb.github.io/mongo-java-driver/4.3/apidocs/mongodb-driver-core/com/mongodb/client/model/WindowedComputations.html) class with factory methods for supported window operators.

Reply:
Great idea!